eth-types/src/geth_types.rs

```diff
@@ -52,14 +52,24 @@ impl TxType {
         matches!(*self, Self::L1Msg)
     }
 
-    /// If this type is EIP-155 or not
+    /// If this type is PreEip155
+    pub fn is_pre_eip155(&self) -> bool {
+        matches!(*self, TxType::PreEip155)
+    }
+
+    /// If this type is EIP155 or not
     pub fn is_eip155_tx(&self) -> bool {
```

Comment:
nitpick. Based on the new nomenclature for the methods here, this should be:
```suggestion
    pub fn is_eip155(&self) -> bool {
```

Reply:
Renamed.